src/pbcontroller.cpp

```diff
@@ -177,9 +177,9 @@ PbController::PbController()
 	}
 
 	config_file =
-		config_dir + std::string(NEWSBEUTER_PATH_SEP) + config_file;
-	queue_file = config_dir + std::string(NEWSBEUTER_PATH_SEP) + queue_file;
-	lock_file = config_dir + std::string(NEWSBEUTER_PATH_SEP) + lock_file;
+		config_dir + NEWSBEUTER_PATH_SEP + config_file;
```

Comment:
Looks like this statement is short enough to fit into a single line now, so please remove the newline after `=`.